content/00.front-matter.md

```diff
@@ -4,9 +4,21 @@
 
 {% for author in authors %}
 + **{{author.full_name}}**<br>
-  [\@{{author.github_username}}](https://github.com/{{author.github_username}}) ·
-  [{{author.orcid}}](https://orcid.org/{{author.orcid}}) ·
-  {{author.affiliations}} ·
-  Funded by {{author.funders}}
+  {% set previous = false %}
```

Comment:
Maybe just assume (but do not enforce) that GitHub username will always be there. A leading interpunct is not the end of the world. Or you could switch to ORCID first and assume its always present. This way you'd incent everyone to have an ORCID.